### ISSUE TITLE
Perftest, where?

### ISSUE THREAD

**Issue by oleg68** [GrandOrgue/grandorgue#868]
### Discussed in https://github.com/GrandOrgue/grandorgue/discussions/867

<div type='discussions-op-text'>

<sup>Originally posted by **jason-barba** November 18, 2021</sup>
With regard to https://github.com/GrandOrgue/grandorgue/discussions/862 and other threads, where can I find the perftest application? My OS i Ubuntu 20.04 LTS.</div>

**Comment by ahall41** [comment]
_make perftest_ gives an error:
```
.../grandorgue/src/tools/perftest.cpp:14:10: fatal error: GODefinitionFile.h: No such file or directory
   14 | #include "GODefinitionFile.h"
```

**Comment by oleg68** [comment]
Fixed with #1076
